docs/Releasing.md

```diff
@@ -4,16 +4,38 @@ Open Enclave SDK Release Procedures
 This document covers how we handle creating a release. Let's look at it from an
 example:
 
+Selecting a Release Manager
+---------------------------
+
+A committer can be nominated to become the release manager by drafting a pull
+request that sets the "Release" tag to the "Area" of one of the committers in
+[Committers.md](Committers.md). The submitter of that nomination should then
+send a link to that pull request through email to all of the committers for open
+discussion. This discussion & PR should be open for at least 3 days, and people
+can make their votes known in the pull request itself by either approving the
+pull request or requesting changes. If there is any opposition from the
+committers after 3 days, then the Committee Governance Committee should take up
+this issue as high priority over email and decide whether to accept or reject
+the new release manager based on feedback from the other committers. If there is
+no opposition from the committers after three days, the newly nominated
+committer becomes the release manager and the pull request to add the "Release"
+tag to this committer gets merged to master. Only committers can nominate other
```

Comment:
This implies no self-nomination and if there is a voting process in place for opposition, it might be unnecessary.

Reply:
Ah thank you. Let me remove "other" here :)